pkg/fixtures/writer_mysql_plain.go

```diff
@@ -50,50 +49,100 @@ func NewMysqlPlainFixtureWriterWithInterfaces(logger log.Logger, client db.Clien
 }
 
 func (m *mysqlPlainFixtureWriter) Purge(ctx context.Context) error {
-	err := m.purger.purgeMysql(ctx)
+	err := m.purger.Purge(ctx)
 	if err != nil {
 		m.logger.Error("error occurred during purging of table %s in plain mysql fixture loader: %w", m.metadata.TableName, err)
 
-		return err
+		return fmt.Errorf("failed to purge mysql table %s: %w", m.metadata.TableName, err)
 	}
 
 	m.logger.Info("purged table %s for plain mysql fixtureSets", m.metadata.TableName)
 
 	return nil
 }
 
-func (m *mysqlPlainFixtureWriter) Write(ctx context.Context, fs *FixtureSet) error {
-	for _, item := range fs.Fixtures {
-		fixture := item.(MysqlPlainFixtureValues)
+func (m *mysqlPlainFixtureWriter) buildSql(fixtures []any) (stmts []string, argss [][]any, err error) {
+	cols := make([]string, len(m.metadata.Columns))
+	for i, col := range m.metadata.Columns {
+		cols[i] = fmt.Sprintf("`%s`", col) // need to quote these, as they might contain mysql keywords
+	}
 
-		sql, args, err := m.buildSql(fixture)
-		if err != nil {
-			return err
+	insertBuilder := squirrel.
+		Replace(m.metadata.TableName).
+		PlaceholderFormat(squirrel.Question).
+		Columns(cols...)
+	offset := 0
+	var stmt string
+	var args []any
+
+	for i, values := range fixtures {
+		// if we exceed the max amount of parameters for a prepared statement, make it a new statement instead
+		if ((i+1)*len(cols))-offset > 65536 {
```

Comment:
magic number? move it into a const?

Reply:
done that. It's the max amount of prepared statement arguments mysql allows you to use.